superflore/generators/bitbake/ros_meta.py

```diff
@@ -12,67 +12,63 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
-# TODO(allenh1): remove inheritance like the RosOverlay class.
+import os
 import random
 import string
 import time
 
-from superflore import RepoInstance as repo_instance
+from superflore.generators.ebuild.overlay_instance import get_random_tmp_dir
```

Comment:
This looks like a chance for better sharing by moving to utils?

Reply:
Yes, this was a consideration... I think I'll change it to have a function that returns a group of random characters of some length. That way I could just call it and inline the formatted directories.

Reply:
Sorry looking under the hood you should be using tempfile.mkdtemp And while you're doing that use a context manager to clean it afterwards. like this: https://gist.github.com/tfoote/f68ad3f6a6a04a41264f65a04ef86fac